docs/modules/ROOT/pages/index.adoc

```diff
@@ -32,3 +32,20 @@ You can xref:user-guide/defining-camel-routes.adoc[define Camel routes] using th
 * xref:user-guide/index.adoc[User guide]
 * xref:contributor-guide/index.adoc[Contributor guide]
 * xref:reference/index.adoc[Extensions reference]
+
+== Compatibility Matrix
+
+[caption=]
+.Compatibility between Camel-Quarkus and dependencies
```

Comment:
Nitpick - Please can we stick with the unhyphenated `Camel Quarkus`. Same for line 42.

Reply:
Yes, good idea.